backend/tests/test_state_transitions.py

```diff
@@ -0,0 +1,70 @@
+from copy import deepcopy
+from datetime import datetime
+from pathlib import Path
+from unittest.mock import patch
+
+import pytest
+from program.media import Show, Season, Episode, Movie
+from program.state_transition import process_event
+from program.libaries import SymlinkLibrary
+from program.settings.manager import settings_manager
+from program.media.state import States
+from program.indexers import TraktIndexer
+
+
+@pytest.fixture
+def test_show():
+    return Show({'imdb_id': 'tt1405406'})
+
+@pytest.fixture
+def test_library_items():
+    settings_manager.settings.symlink.library_path = Path(__file__).parent / "library"
+    library = SymlinkLibrary()
+    return [item for item in library.run()]
+
+def test_symlink_library_output(test_library_items):
+    assert all(i.state == States.Completed for i in test_library_items)
+    assert len([i for i in test_library_items if isinstance(i, Movie)]) == 5
+    assert len([i for i in test_library_items if isinstance(i, Show)]) == 1
+    show = next(i for i in test_library_items if isinstance(i, Show))
+    assert len(show.seasons) == 8
+
+def test_library_to_index(test_show):
+    media_item, service, output_items = process_event(None, SymlinkLibrary, test_show)
+    assert service == TraktIndexer
```

Comment:
What are we really testing here? Split this up and rename the tests to something more meaningful.